src/Microsoft.ML.StandardLearners/Standard/MultiClass/OneVersusAllTrainer.cs

```diff
@@ -62,30 +62,27 @@ internal sealed class Options : OptionsBase
         }
 
         /// <summary>
-        /// Legacy constructor that builds the <see cref="Ova"/> trainer supplying the base trainer to use, for the classification task
-        /// through the <see cref="Options"/>.
-        /// Developers should instantiate OVA by supplying the trainer argument directly to the OVA constructor
-        /// using the other public constructor.
+        /// Constructs a <see cref="OneVersusAllTrainer"/> trainer supplying a <see cref="Options"/>.
         /// </summary>
         /// <param name="env">The private <see cref="IHostEnvironment"/> for this estimator.</param>
         /// <param name="options">The legacy <see cref="Options"/></param>
-        internal Ova(IHostEnvironment env, Options options)
+        internal OneVersusAllTrainer(IHostEnvironment env, Options options)
             : base(env, options, LoadNameValue)
         {
             _options = options;
         }
 
         /// <summary>
-        /// Initializes a new instance of <see cref="Ova"/>.
+        /// Initializes a new instance of <see cref="OneVersusAllTrainer"/>.
         /// </summary>
         /// <param name="env">The <see cref="IHostEnvironment"/> instance.</param>
         /// <param name="binaryEstimator">An instance of a binary <see cref="ITrainerEstimator{TTransformer, TPredictor}"/> used as the base trainer.</param>
-        /// <param name="calibrator">The calibrator. If a calibrator is not explicitely provided, it will default to <see cref="PlattCalibratorTrainer"/></param>
+        /// <param name="calibrator">The calibrator. If a calibrator is not provided, it will default to <see cref="PlattCalibratorTrainer"/></param>
```

Comment:
How does a naturally-calibrated model interact with this parameter?

Reply:
I dont know for sure - I will posted an issue as a question. #2805

---
In reply to: [261015949](https://github.com/dotnet/machinelearning/pull/2743#discussion_r261015949) [](ancestors = 261015949)

Reply:
Also I think it would be worth updating this comment once I have more info. I won't that block checking this in for now though.

---
In reply to: [261430492](https://github.com/dotnet/machinelearning/pull/2743#discussion_r261430492) [](ancestors = 261430492,261015949)